packages/server/src/base/streamer.ts

```diff
@@ -67,15 +67,14 @@ export class Streamer {
       await this.dispatcher.publish(StreamerDispatches.Message, userId, { source: source?.socket?.id, payload })
     }
 
-    if (source?.client?.id !== clientId) {
-      const webhooks = await this.webhooks.list(clientId)
-
-      for (const webhook of webhooks) {
-        void this.send(process.env.SPINNED_URL || webhook.url, payload, {
-          'x-bp-messaging-client-id': clientId,
-          'x-bp-messaging-webhook-token': webhook.token
-        })
-      }
+    // TODO: this might break stuff on botpress's side
+    const webhooks = await this.webhooks.list(clientId)
+
+    for (const webhook of webhooks) {
+      void this.send(process.env.SPINNED_URL || webhook.url, payload, {
+        'x-bp-messaging-client-id': clientId,
+        'x-bp-messaging-webhook-token': webhook.token
+      })
     }
```

Comment:
I think we'll have to ignore bot message on botpress

Reply:
And why is that?

Reply:
We don't want the bot reacting to messages it just posted

Reply:
Then why not extend the mechanism that tells the source of a message so that we know not to send the event?

Reply:
We'll have that with more complex webhook settings (only send incoming/outgoing, only send for x channel, etc). To be done in future PR